nix/rollup-bridge-contracts.nix

```diff
@@ -45,8 +45,11 @@ stdenv.mkDerivation rec {
     npx hardhat clean && npx hardhat compile
   '';
 
+  doCheck = enableTesting;
```

Comment:
seems it useless till we have no unit tests

Reply:
fixed